### PR TITLE
remove gone but not forgotten script

### PR DESCRIPTION
**Pull request by northofnormal**

### CODE REVIEW

File: scripts/gbnf.coffee [hunk 1]
```diff
@@ -1,46 +0,0 @@
...
-  "http://i.imgur.com/zAY3NrR.jpg",
-  "http://i.imgur.com/IHP0ZAH.jpg"
-]
-
-initBrain  = (robot) ->
-  if not robot.brain.get(gbnf_images_key)?
-    robot.brain.set gbnf_images_key, default_gbnf_images
-
-module.exports = (robot) ->
-  robot.hear /gone (but|and) not forgotten/i, (msg) ->
```
Comment (macklinu) [inline scripts/gbnf.coffee]: If we would still like to remember former Labbers, we could change this from `robot.hear` to `robot.respond`. I rarely interact with this script, so I don't know if it's valuable to the team, but that's just a thought I had. Could go either way.

Comment (nwest) [inline scripts/gbnf.coffee]: I'm cool with that change

Comment (northofnormal) [inline scripts/gbnf.coffee]: Me, too--this was one of the ones I was thinking about recreating as a `/command` in Slack.

Comment (nwest) [inline scripts/gbnf.coffee]: Great idea!

Comment (macklinu) [inline scripts/gbnf.coffee]: I can get this PR merged if this is something you'd like to migrate to a Slack `/command`, @northofnormal.

**Pull request closed without merging**